packages/blocks/src/root-block/edgeless/edgeless-root-block.ts

```diff
@@ -765,10 +766,9 @@ export class EdgelessRootBlockComponent extends BlockElement<
     );
   }
   private _isPieMenuOpen() {
-    return false; // TODO add it back later
-    // return (
-    //   this.widgetElements['affine-pie-menu-widget'] as AffinePieMenuWidget
-    // ).isOpen;
+    return (
+      this.widgetElements['affine-pie-menu-widget'] as AffinePieMenuWidget
+    ).isOpen;
```

Comment:
I don't think this is necessary. Looks like you're using this method to determine whether to execute the wheel event handler. Am I right?
Since the widget is rendered inside the `edgeless-root-block`, you could just call `e.stopPropagation()` in the pie-menu widget wheel handler, preventing the event from propagating to the upper element.